CONTRIBUTORS.md

```diff
@@ -15,3 +15,4 @@ Cerner Corporation
 [@tbiethman]: https://github.com/tbiethman
 [@mjhenkes]: https://github.com/mjhenkes
 [@nagrawal3]: https://github.com/nagrawal3
+[@kschuste]: https://github.com/kschuste
```

Comment:
You'll want to add your name above, your name doesn't show up on the view only mark down file

Reply:
done